docs/user_guide/assessment.rst

```diff
@@ -99,11 +226,12 @@ We then calculate a metric which shows the subgroups:
     >>> print("recall by groups = ", grouped_metric.by_group.to_dict())
     recall by groups =  {'a': 0.0, 'b': 0.5, 'c': 0.75, 'd': 0.0}
 
-Note that the overall recall is the same as that calculated above in the
-Ungrouped Metric section, while the 'by group' dictionary can be checked
-against the table above.
+The disaggregated metrics are stored in a pandas Series 
```

Comment:
I'd suggest using intersphinx for pandas.Series using :class:

Reply:
Done!